AssetRipperCore/Classes/Shader/Shader.cs

```diff
@@ -222,11 +270,25 @@ private void UnpackSubProgramBlobs(LayoutInfo layout, uint[][] offsets, uint[][]
 		}
 
 		public override string ExportExtension => "shader";
+		public override string Name => this.GetValidShaderName();
```

Comment:
Why were you wanting to change this?

Reply:
To properly show the name in GUI and console. I have a different approach in `Shader.cs` from #304